stubs/workalendar/workalendar/core.pyi

```diff
@@ -0,0 +1,206 @@
+from _typeshed import Incomplete
+from collections.abc import Generator
+
+MON: Incomplete
+TUE: Incomplete
+WED: Incomplete
+THU: Incomplete
+FRI: Incomplete
+SAT: Incomplete
+SUN: Incomplete
+ISO_MON: Incomplete
+ISO_TUE: Incomplete
+ISO_WED: Incomplete
+ISO_THU: Incomplete
+ISO_FRI: Incomplete
+ISO_SAT: Incomplete
+ISO_SUN: Incomplete
+
+class classproperty:
```

Comment:
Presumably this should be removed. In the stubs we could just use `ClassVar` for anything accessible through a class.